.cspell.json

```diff
@@ -0,0 +1,85 @@
+{
+  "version": "0.1",
+  "language": "en",
+  "ignorePaths": [
+    "**/coverage/**",
+    "**/node_modules/**",
+    "**/dist/**",
+    "**/fixtures/**",
+    "**/tests/**",
+    "**/**/CHANGELOG.md",
+    "**/**/CONTRIBUTORS.md",
+    "**/**/ROADMAP.md",
+    "**/*.d.ts"
+  ],
+  "dictionaries": [
+    "typescript",
+    "softwareTerms",
+    "node",
+    "en_US",
+    "npm",
+    "misc",
+    "filetypes"
+  ],
+  "ignoreRegExpList": [
+    "/```[\\w\\W]*?```/",
```

Comment:
Should this regular expression account for newline after the opening fence and language identifier, and newline and optional whitespace before the closing fence, to be more accurate?

Reply:
i think adding should be sufficient, unless you have other idea?
```
    "/```[\\w\\W]*?```/",
    "/~~~[\\w\\W]*?~~~/",
    "/``[\\w\\W]*?``/",
    "/`[^`]*`/",
```